satpy/tests/test_demo.py

```diff
@@ -171,3 +172,23 @@ def test_no_gcsfs(self):
         """Test that 'gcsfs' is required."""
         from satpy.demo._google_cloud_platform import get_bucket_files
         self.assertRaises(RuntimeError, get_bucket_files, '*.nc', '.')
+
+
+class TestAHIDemoDownload(unittest.TestCase):
```

Comment:
No need to make this a unittest subclass, right? I personally prefer the no base class option so we can use things like pytest parametrize in the future.